CMakeLists.txt

```diff
@@ -508,15 +515,15 @@ if(AVIF_CODEC_RAV1E)
     else()
         # Check to see if rav1e is independently being built by the outer CMake project
         if(NOT TARGET rav1e)
-            find_package(rav1e REQUIRED)
+            find_package_libavif(rav1e REQUIRED)
             set(AVIF_CODEC_SYSTEM_INCLUDES ${AVIF_CODEC_SYSTEM_INCLUDES} ${RAV1E_INCLUDE_DIR})
         endif()
         set(AVIF_CODEC_LIBRARIES ${AVIF_CODEC_LIBRARIES} ${RAV1E_LIBRARIES})
     endif()
 
     # Unfortunately, rav1e requires a few more libraries
     if(WIN32)
-        set(AVIF_PLATFORM_LIBRARIES ${AVIF_PLATFORM_LIBRARIES} ws2_32.lib bcrypt.lib userenv.lib ntdll.lib)
+        set(AVIF_PLATFORM_LIBRARIES ${AVIF_PLATFORM_LIBRARIES} legacy_stdio_definitions.lib ntdll.lib advapi32.lib userenv.lib ws2_32.lib bcrypt.lib msvcrt.lib)
```

Comment:
@fdintino Why were legacy_stdio_definitions.lib, advapi32.lib, and msvcrt.lib added?

Reply:
I had missing symbols errors without them. I got this list from the windows rav1e.pc Libs.private